onsp_co2_purchase/__manifest__.py

```diff
@@ -9,6 +9,7 @@
     "depends": [
         "onsp_co2",
         "purchase",
+        "contacts",
```

Comment:
Why do you need contacts?

Reply:
Pretty sure so, the onsp_co2 used the model res.partner so i though we'll need it...

Reply:
res.partner is a model declared in the base module... :wink:

Reply:
Didn't thanks for that... fix in https://github.com/MyCityCO2/mycityco2-engine/pull/13/commits/8b2e02ceb9c4ff9b92493a366cca5e58f01e1257